packages/webpack/src/config/modern.js

```diff
@@ -0,0 +1,31 @@
+import _ from 'lodash'
```

Comment:
Please use single imports for lodash :raised_hands:

Reply:
Good catch